okhttp/src/main/java/com/squareup/okhttp/internal/DiskLruCache.java

```diff
@@ -206,29 +214,31 @@ void initialize() throws IOException {
       try {
         readJournal();
         processJournal();
+        initialized = true;
```

Comment:
I hate initializing this in two places and having to reset `closed` below. This method's flow control is wonky.